index.html

```diff
@@ -10,12 +10,16 @@
     <body>
         <form id="todoInput">
             <label for="inputField">Add ToDo:</label>
-            <input id="inputField" name="inputField" type="text">
-            <button type="submit">+</button>
+            <input id="inputField" name="inputField" type="text" value="">
+            <button  id="addBtn" type="submit">+</button><br>
```

Comment:
Please make sure there are no extra spaces in the between the attributes of html elements
```<button  id="addBtn" type="submit">+</button>```
==>
```<button id="addBtn" type="submit">+</button>```
(ONLY One Space after `<button`)